Content.Client/Storage/StorageBoundUserInterface.cs

```diff
@@ -61,9 +61,9 @@ public void BuildEntityList(EntityUid uid, StorageComponent component)
             _window?.BuildEntityList(uid, component);
         }
 
-        public void InteractWithItem(BaseButton.ButtonEventArgs args, ListData cData)
+        public void InteractWithItem(BaseButton.ButtonEventArgs? args, ListData? cData)
         {
-            if (cData is not EntityListData { Uid: var entity })
+            if (args == null || cData is not EntityListData { Uid: var entity })
```

Comment:
... since this doesn't do anything.

Reply:
This change is mostly for the PlayerListControl so that it can deselect a player and disable buttons like kick and teleport when repopulating.

It doesn't do anything on storage right now. Storage did need some changes to be able to handle the changed method though.